### PR TITLE
FUSETOOLS-2891: Adapt UI tests to changes made in new Fuse project wizard

### PR DESCRIPTION
**Pull request by apodhrad**
…zard

Signed-off-by: Andrej Podhradsky <apodhrad@redhat.com>

# Pull Request Checklist

## General

- [x] Did you use the Jira Issue number in the commit comments?
- [x] Did you set meaningful commit comments on each commit?
- [x] Did you sign off all commits for this PR? (git commit -s -m "jira issue number - your commit comment")

## Function

- [ ] Does the project still build fine locally?
- [ ] Does your modification work?
- [ ] Is the non-happy path working, too?
- [ ] Are other parts that use the same component still working fine?

## Code Style

- [ ] Are method-/class-/variable-names meaningful?
- [ ] Are methods concise, not too long?
- [ ] Are catch blocks catching precise Exceptions only (no catch all)?
- [ ] Have you used the correct file header copyright comment?
- [ ] Have you set the correct year in the headers of new files?

## Tests

- [ ] Are there unit-tests?
- [ ] Are there integration tests (or at least a jira to tackle these)?

### CODE REVIEW

**Comment by apodhrad** [comment]
This PR can be merged after merging https://github.com/jbosstools/jbosstools-fuse/pull/1310

**Comment by apodhrad** [comment]
testPR

**Review by apupier — DISMISSED** [review]

**Review by tsedmik — APPROVED** [review]

**Pull request merged**
